notebooks/daniel_notebook.ipynb

```diff
@@ -1,191 +1,26 @@
 {
```

Comment:
<ul><li><code>norm</code>  is a measure of vector magnitude. </li><li>use the "`"-ticks to show that these are functions</li><li><code>plt.figure</code>  creates a figure, the Axes3D will give 3d axes</li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>